test/runner.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+const wptRunner = require("..");
+
+const testcases = require("./testcases.json");
+const filter = testPath => (testcases[testPath] === true);
```

Comment:
I'd rather just enumerate all the files in the test/tests directory using the fs APIs; then we don't need to manually update a testcases.json file.